pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -97368,8 +97368,6 @@ self: {
        ];
        description = "The core of an IDE";
        license = stdenv.lib.licenses.asl20;
-       hydraPlatforms = stdenv.lib.platforms.none;
-       broken = true;
```

Comment:
When you're sending a PR to `haskell-updates`, can you be sure not to edit the `hackage-packages.nix` file?

This is to reduce merge conflicts when it is updated automatically.

This post has a couple small pieces of advice when sending PRs to haskell updates:

https://discourse.nixos.org/t/call-to-action-for-updating-haskell-packages-after-bump-to-lts-15/6071

Reply:
Okay, I uploaded a new version without any changes to `haskell-packages.nix`.

But now you can‘t test it …

Reply:
Maybe I should have trodden the middleground there? Let me know what you think.